pytest_fixtures/component/taxonomy.py

```diff
@@ -262,3 +280,10 @@ def upgrade_entitlement_manifest():
     upgrade scenarios"""
     manifestor = Manifester(manifest_category=settings.manifest.entitlement)
     return manifestor.get_manifest(), manifestor
+
+
+@pytest.fixture
+def sca_manifest_for_upgrade():
```

Comment:
would it be ok to change the name something like `fun_sca_manifest_for_upgrade` or `function_manifest_for_upgrade` for better understanding of fixture scope

Reply:
I already changed the name for @jyejare. Lets keep it as is